frontend/src/app/offers/offer-detail/offer-detail.component.ts

```diff
@@ -24,7 +24,12 @@ export class OfferDetailComponent implements OnInit {
 
   ngOnInit() {
     this.offer$ = this.activatedRoute.params
-    .switchMap(params => this.offersService.getOffer(params.offerId));
+    .switchMap(params => this.offersService.getOffer(params.offerId))
+    .map(offer$ => {
```

Comment:
`$` at the end means observable

Reply:
if we will move this logic to service it will be available for all components instaed of only that one

Reply:
getOffer(id) method returns an observable so I thought that notation with $ would be correct in this case.
I moved this logic to the service as a loadDefaultImage method, and I used it in getOffer and getOffers methods to load the volontulo banner if there is no image for an offer. Now there are no empty images not only on offer-details pages but also on home page.